pkg/consts/consts.go

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package consts
+
+const (
+	BundleVersionAnnotation = "gateway.networking.k8s.io/bundle-version"
+	ChannelAnnotation       = "gateway.networking.k8s.io/channel"
```

Comment:
Public symbols should have `godoc` comments

Reply:
oh right 👍

Reply:
Comments added